accounting/entries.go

```diff
@@ -231,15 +231,60 @@ func sweepEntries(tx lndclient.Transaction, u entryUtils) ([]*HarmonyEntry, erro
 	return []*HarmonyEntry{txEntry, feeEntry}, nil
 }
 
+// isUtxoManagementTx checks whether a transaction is restructuring our utxos.
+func isUtxoManagementTx(txn lndclient.Transaction) bool {
+	// Check all inputs.
+	for _, input := range txn.PreviousOutpoints {
+		if !input.IsOurOutput {
+			return false
+		}
+	}
+
+	// Check all outputs.
+	for _, output := range txn.OutputDetails {
+		if !output.IsOurAddress {
+			return false
+		}
+	}
+
+	// If all inputs and outputs belong to our wallet, it's utxo management.
+	return true
+}
+
+// createOnchainFeeEntry creates a fee entry for an on chain transaction.
+func createOnchainFeeEntry(tx lndclient.Transaction, category string, note string, u entryUtils) (*HarmonyEntry, error) {
```

Comment:
nit: line length (we're trying to adhere to the same code formatting rules as [in `lnd`](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#80-character-line-length), including the 80 character line length limit.

Reply:
Updated.